src/cowboy.appup.src

```diff
@@ -1,8 +1,8 @@
 %% -*-: erlang -*-
 
-{"2.8.3",
+{"2.8.4",
   [
-    {"2.8.2", [
+    {<<"2\\.8\\.[23]">>, [
```

Comment:
2-3?

Reply:
Yes, in this particular case it's the same :)